docs/index.md

```diff
@@ -1,64 +1,27 @@
 
-  script = <<-END
-    #!/bin/bash
-    echo "hello!"
-  END
-}
-```
+- **Provision Resources**: create cloud compute (CPU, GPU, RAM) & storage resources without reading pages of documentation
+- **Sync & Execute**: easily sync & run local data & code in the cloud
+- **Low cost**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
+- **No waste**: auto-cleanup unused resources (terminate compute instances upon job completion/failure & remove storage upon download of results)
+- **No lock-in**: switch between several cloud vendors with ease due to concise unified configuration
 
--> **Note:** See [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) for more information.
+Supported cloud vendors include:
 
-## Authentication
+- Amazon Web Services (AWS)
+- Microsoft Azure
+- Google Cloud Platform (GCP)
+- Kubernetes (K8s)
 
-Environment variables are the only supported authentication method. They should be present when running any of the `terraform` commands.
+## Links
 
-### Example
-
-```bash
-$ export GOOGLE_APPLICATION_CREDENTIALS_DATA="$(cat service_account.json)"
-$ terraform apply
-```
-
-### Amazon Web Services
-
-- `AWS_ACCESS_KEY_ID` - Access key identifier.
-- `AWS_SECRET_ACCESS_KEY` - Secret access key.
-- `AWS_SESSION_TOKEN` - (Optional) Session token.
-
-See the [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) for more information.
-
-### Microsoft Azure
-
-- `AZURE_CLIENT_ID` - Client identifier.
-- `AZURE_CLIENT_SECRET` - Client secret.
-- `AZURE_SUBSCRIPTION_ID` - Subscription identifier.
-- `AZURE_TENANT_ID` - Tenant identifier.
-
-See the [Azure documentation](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.environmentcredential) for more information.
-
-### Google Cloud Platform
-
-- `GOOGLE_APPLICATION_CREDENTIALS` - Path to a service account JSON key file.
-
--> **Note:** you can also use `GOOGLE_APPLICATION_CREDENTIALS_DATA` with the **contents** of the service account JSON key file.
-
-See the [GCP documentation](https://cloud.google.com/docs/authentication/getting-started#creating_a_service_account) for more information.
-
-### Kubernetes
-
-- `KUBECONFIG` - Path to a [`kubeconfig` file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable).
-
--> **Note:** You can use `KUBECONFIG_DATA` instead, with the **contents** of the `kubeconfig` file.
+- [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started)
+  - [Authentication](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/authentication)
+- [Full reference](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task)
```

Comment:
These links are in a special arrangement different from the left-side content structure. Causes confusion as to what's more important to read next I think. Maybe I don't understand the purpose of the **Links** section?

Reply:
p.s. also these links open in a separate window unexpectedly.